Cloudflare/gateway_http/ingest/parser.yml

```
@@ -32,10 +32,17 @@ stages:
       - set:
           event.type: [ "denied" ]
         filter: '{{json_event.message.Action == "deny"}}'
+      - set:
+          file.hash.sha256: '{{json_event.message.BlockedFileHash}}'
+        filter: '{{json_event.message.BlockedFileHash is not in [""]}}'
```

```suggestion
        filter: '{{ json_event.message.BlockedFileHash | length == 32 }}'
```
As you suggested, I think this check will be enough

---

`is not in [""]` filter isn't needed for ECS fields, only for custom ones.